_data/changelogs/component-form.yml

```
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed high-contrast disabled border from text input standard variants.
```

Last release, we changed all "high-contrast" references to "forced colors". Added a couple suggestions for structure and to suggest that the current disabled styles were not intentional: 
```suggestion
    summary: Fixed a bug that caused standard text input variants to show disabled styles when in forced colors mode.
```